plugins/de.ovgu.featureide.fm.core/src/de/ovgu/featureide/fm/core/io/xml/XmlFeatureModelFormat.java

```diff
@@ -454,6 +454,9 @@ private void parseAttribute(Element e, LinkedList<FeatureAttribute> attributeLis
 
 			} else {
 
+				if (checkAttributeList(name, recursiveList)) {
+					throwError("There already is an inherited FeatureAttribute with that name.", e);
```

Comment:
throwError("There already is an inherited FeatureAttribute with that name.", e);  ->>
There is already